site/go.mod

```diff
@@ -1,5 +1,8 @@
 module github.com/kubernetes-sigs/prow
 
-go 1.20
+go 1.23.4
```

Comment:
10:08:22 AM: hugo --enableGitInfo --buildFuture -b https://deploy-preview-350--k8s-prow.netlify.app/
10:08:22 AM: go: errors parsing go.mod:
10:08:22 AM: /opt/build/repo/site/go.mod:3: invalid go version '1.23.4': must match format 1.23
10:08:22 AM: Error: failed to download modules: failed to execute 'go [mod download]': failed to execute binary "go" with args [mod download]: go: errors parsing go.mod:
10:08:22 AM: /opt/build/repo/site/go.mod:3: invalid go version '1.23.4': must match format 1.23
10:08:22 AM:  *errors.errorString